Exiled.API/Features/Lift.cs

```diff
@@ -236,6 +232,39 @@ public static bool TryMeltPlayer(Player player)
         /// <returns><see langword="true"/> if the lift was started successfully; otherwise, <see langword="false"/>.</returns>
         public bool TryStart(int level, bool isForced = false) => TrySetDestination(Base.AssignedGroup, level, isForced);
 
+        /// <summary>
+        /// Changes lock of the lift.
+        /// </summary>
+        /// <param name="lockReason">Type of lift lockdown.</param>
+        public void ChangeLock(DoorLockReason lockReason)
+        {
+            DoorLockReason lockFlags = Base.ActiveLocks;
+
+            if (lockReason == DoorLockReason.None)
+                lockFlags = 0;
+            else
+                lockFlags |= lockReason;
+
+            List<ElevatorDoor> targetDoors = ElevatorDoor.AllElevatorDoors.First(x => x.Key == Base.AssignedGroup).Value;
```

Comment:
This should become a property that gets populated with Exiled doors in the constructor, something like `IReadOnlyCollection<Door> Doors`.